pkgs/development/libraries/science/math/tiny-cuda-nn/default.nix

```diff
@@ -0,0 +1,155 @@
+{ cmake
+, fetchFromGitHub
+, lib
+, ninja
+, pythonPackages
+, pythonSupport ? false
+, stdenv
+, symlinkJoin
+, which
+}:
+assert lib.asserts.assertMsg
+  (pythonPackages.torch.cudaSupport)
+  "tiny-cuda-nn requires torch to be built with cudaSupport"; let
```

Comment:
i'm still confused why these are on the same line. wouldn't it be cleaner to have the `let` on a separate line?

Reply:
On closer inspection (and because of my growing discontent with assert statements) it's redundant -- `broken` is set to `!cudaSupport`, and `cudaSupport` is inherited from `torch` so it should be clear that we require a torch compiled with support for CUDA.